packages/lab/src/multiline-input/MultilineInput.css

```diff
@@ -98,14 +98,16 @@
   color: inherit;
   cursor: inherit;
   display: block;
+  height: calc(var(--saltMultilineInput-rows, 3) * var(--salt-text-lineHeight));
   flex: 1;
   font: inherit;
   letter-spacing: 0;
   margin: 0;
   min-width: 0;
   overflow: hidden;
   resize: none;
-  padding: var(--salt-spacing-100) 0;
+  padding: 0;
+  margin: var(--salt-spacing-100) 0;
```

Comment:
you already had margin in 105, you can probably remove it

Reply:
fixed